packages/hint-doctype/tests/tests.ts

```diff
@@ -0,0 +1,238 @@
+import { getHintPath } from 'hint/dist/src/lib/utils/hint-helpers';
+import { HintTest } from '@hint/utils-tests-helpers/dist/src/hint-test-type';
+import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
+
+const hintPath = getHintPath(__filename);
+
+const tests: Array<HintTest> = [
```

Comment:
Nit: we recently cut over to using the `[]` syntax for `Array` types.
```suggestion
const tests: HintTest[] = [
```